var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -125,26 +128,39 @@ def write_makefile_inc(self):
         fpic = '-fPIC' if '+shared' in self.spec else ''
```

Comment:
since you modify it, please change `'-fPIC`to the call of the compiler function, as in previous PR